src/tests/functional/plugin/gpu/shared_tests_instances/single_layer_tests/experimental_detectron_prior_grid_generator.cpp

```diff
@@ -0,0 +1,83 @@
+//
+
+#include <single_layer_tests/experimental_detectron_prior_grid_generator.hpp>
+#include <common_test_utils/data_utils.hpp>
+#include <common_test_utils/ov_tensor_utils.hpp>
+
+namespace {
+
+const std::initializer_list<ov::test::subgraph::ExperimentalDetectronPriorGridGeneratorTestParam> params {
+        // flatten = true (output tensor is 2D)
+        {
+                {true, 0, 0, 4.0f, 4.0f},
+                ov::test::static_shapes_to_test_representation({{3, 4}, {1, 16, 4, 5}, {1, 3, 100, 200}})
+        },
+        // task #72587
+        //{
+        //        {true, 3, 6, 64.0f, 64.0f},
+        //        ov::test::static_shapes_to_test_representation({{3, 4}, {1, 16, 100, 100}, {1, 3, 100, 200}})
+        //},
+        //// flatten = false (output tensor is 4D)
+        {
+                {false, 0, 0, 8.0f, 8.0f},
+                ov::test::static_shapes_to_test_representation({{3, 4}, {1, 16, 3, 7}, {1, 3, 100, 200}})
+        },
+        // task #72587
+        //{
+        //        {false, 5, 3, 32.0f, 32.0f},
+        //        ov::test::static_shapes_to_test_representation({{3, 4}, {1, 16, 100, 100}, {1, 3, 100, 200}})
+        //},
+};
+
+const std::initializer_list<std::pair<std::string, std::vector<ov::Tensor>>> inputTensors {
+        {
+                "test#1",
+                {
+                        ov::test::utils::create_tensor<float>(
+                                ov::element::f32,
+                                {3, 4},
+                                {-24.5, -12.5, 24.5, 12.5, -16.5, -16.5, 16.5, 16.5, -12.5, -24.5, 12.5, 24.5})
+                }
+        },
+        {
+                "test#2",
+                {
+                        ov::test::utils::create_tensor<float>(
+                                ov::element::f32,
+                                {3, 4},
+                                {-44.5, -24.5, 44.5, 24.5, -32.5, -32.5, 32.5, 32.5, -24.5, -44.5, 24.5, 44.5})
+                }
+        },
+        {
+                "test#3",
+                {
+                        ov::test::utils::create_tensor<float>(
+                                ov::element::f32,
+                                {3, 4},
+                                {-364.5, -184.5, 364.5, 184.5, -256.5, -256.5, 256.5, 256.5, -180.5, -360.5, 180.5, 360.5})
+                }
+        },
+        {
+                "test#4",
+                {
+                        ov::test::utils::create_tensor<float>(
+                                ov::element::f32,
+                                {3, 4},
+                                {-180.5, -88.5, 180.5, 88.5, -128.5, -128.5, 128.5, 128.5, -92.5, -184.5, 92.5, 184.5})
+                }
+        }
+};
+
+using ov::test::subgraph::ExperimentalDetectronPriorGridGeneratorLayerTest;
+
+INSTANTIATE_TEST_SUITE_P(smoke_ExperimentalDetectronPriorGridGenerator, ExperimentalDetectronPriorGridGeneratorLayerTest,
+                         testing::Combine(
```

Comment:
The op supports any supported floating point. Could you please add test case for fp16?
https://docs.openvino.ai/latest/openvino_docs_ops_detection_ExperimentalDetectronPriorGridGenerator_6.html

Reply:
done

Reply:
Thanks!